### PR TITLE
#163222740 Implements adding of comments on questions [Delivers #163222740]

### PR DESCRIPTION
**Pull request by chikeozulumba**
#### What does this PR do?
With this functionality, the user should be able to comment on a question
#### Description of Task to be completed?
- Add middleware for validating comments
- Setup comment route
- Modify migrations
- Refactor meetup controller 
- Change tests script
- Implement unit tests
#### What are the relevant pivotal tracker stories?
[#163222740](https://www.pivotaltracker.com/story/show/163222740)

### CODE REVIEW

File: server/migrations/comment.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ const Comment = async () => {
           id SERIAL UNIQUE, 
           userid INTEGER NOT NULL, 
           meetup INTEGER NOT NULL,
```
Comment (robocopkaka) [inline server/migrations/comment.js]: Shouldn't this be `meetupId`

File: server/migrations/comment.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ const Comment = async () => {
           id SERIAL UNIQUE, 
           userid INTEGER NOT NULL, 
           meetup INTEGER NOT NULL, 
+          question INTEGER NOT NULL,
```
Comment (robocopkaka) [inline server/migrations/comment.js]: And `questioId`

Comment (chikeozulumba) [reply server/migrations/comment.js]: Noted

File: server/migrations/comment.js (hunk 1 above)
Comment (chikeozulumba) [reply server/migrations/comment.js]: Noted

File: server/migrations/comment.js (hunk 2 above)
Comment (chikeozulumba) [reply server/migrations/comment.js]: I have made the changes

File: server/migrations/comment.js (hunk 1 above)
Comment (chikeozulumba) [reply server/migrations/comment.js]: I have changed the field name

**Pull request merged**